wordpress-mobile/WordPress-iOS
# [Stats Refresh] Remove feature flag

Ref #10975

This removes `statsRefresh` feature flag, associated checks, and all dependencies on `WordPressComStatsiOS`. (Except for `TodayViewController`, which was done in #12767.)

To test:
Basically, verify Stats still works.

- To go Stats. 
  - Verify new Stats appears.
- Set the Today widget.
  - Verify the Today widget uses the correct site.
- Go to 'Blog Posts'.
  - On a post, select 'More' then 'Stats'.
  - Verify 'Post Stats' is displayed.

Update release notes:
